python/hail/expr/functions.py

```diff
@@ -1548,3 +1548,48 @@ def to_str(x):
     :class:`.StringExpression`
     """
     return _func("str", TString(), x)
+
+@typecheck(pl=ArrayInt32Expression)
+def gt_from_pl(pl):
+    """Call genotype from Phred-scaled probability likelihoods.  Returns
```

Comment:
Too long, this will look bad in the rendered doc headers.. Move the second sentence to notes

Reply:
Good to know.  Is there an approximate character limit we should aim for in the first line?